tests/integration/plugins/test_changes.py

```diff
@@ -7,10 +7,21 @@
 pytestmark = pytest.mark.asyncio
 
 
+def find_changes_record(records: list[dict], bucket: str, collection: str):
+    return next(
+        (
+            record
+            for record in records
+            if record["collection"] == collection and record["bucket"] == bucket
+        )
+    )
+
+
 async def test_changes_plugin(
```

Comment:
It's unlikely that we'll run the tests with a different collection than `main-workspace/product-integrity`, but since we offer to specify the collection (eg. `main-workspace/normandy-recipes-capabilities`) then we may want to be smart, yes.
So the test would not be exactly based the to-review-enabled flag passed in CLI. Instead, the test could adjust its behavior based on if the specified `source_collection` has review enabled or not. 

We check this here for example: https://github.com/mozilla-services/remote-settings-lambdas/blob/c14d6f8ab0269c1a578271fbfaefd45f505d2a9f/commands/backport_records.py#L106-L140
(in a very verbose way I admit)

Reply:
Hmm. This smells a little like YAGNI, then? So maybe we can defer on adjusting the test until it's needed and unblock #389.